assets/css/v2/elevator/elevator_closures.scss

```diff
@@ -34,118 +61,238 @@
-        line-height: 112px;
-
-        &__title {
-          word-spacing: 9999px;
+    .header {
+      box-sizing: border-box;
+      height: 528px;
+      padding: 53px 68px 51px 48px;
+      font-weight: 700;
+
+      .icons {
+        margin-bottom: 21px;
+
+        .no-service-icon {
+          margin-right: 16px;
         }
       }
+
+      .closed-text {
+        margin-bottom: 18px;
+        font-size: 150px;
+        line-height: 150px;
+      }
+    }
+
+    hr.thin {
+      margin: 0 20px;
     }
 
-    .closure-list-container {
-      overflow: hidden;
+    .accessible-path-container {
+      box-sizing: border-box;
+      height: 984px;
+      padding: 48px;
 
-      .closure-list {
+      .subheading-container {
         display: flex;
-        flex-flow: column wrap;
-        height: 904px;
-        transform: translateX(calc(-100% * var(--closure-list-offset)));
-
-        .closure-row {
-          box-sizing: border-box;
-          width: 1080px;
-          padding: 0 48px;
+        align-items: center;
+        justify-content: space-between;
+        height: 142px;
 
-          &__station-name {
-            font-size: 62px;
-            font-weight: 600;
-            line-height: 80px;
-            color: $cool-black-15;
-          }
+        .subheading {
+          align-self: center;
+        }
 
-          &__name-and-pills {
-            display: flex;
-            align-items: center;
-            margin-bottom: 14px;
+        .arrow {
+          width: 100px;
+          height: 100px;
+        }
+      }
 
-            .route-pill {
-              width: 132px;
-              height: 68.13px;
-              margin-right: 24px;
+      .alternate-direction-text {
+        color: $cool-black-30;
+      }
 
-              &__text {
-                line-height: 68.13px;
-              }
+      .alternate-direction-text.small {
+        font-size: 48px;
+        line-height: 64px;
+      }
 
-              &__icon {
-                height: 100%;
-              }
+      .alternate-direction-text.medium {
+        font-size: 62px;
+        line-height: 80px;
+      }
 
-              &__icon-image {
-                height: 100%;
-                margin-top: 0;
-              }
-            }
-          }
+      .alternate-direction-text.large {
+        font-size: 80px;
+        line-height: 96px;
+      }
 
-          &__elevator-name {
-            line-height: 64px;
-          }
+      .map-container {
+        position: relative;
+        margin-top: 26px;
+        margin-left: -48px;
 
-          &__elevator-name.list-item {
-            display: list-item;
-            margin-bottom: 8px;
-            margin-left: 48px;
-          }
+        .map {
+          width: 1080px;
         }
       }
     }
 
-    .paging-info-container {
-      position: absolute;
-      bottom: 0;
+    .paging-indicators {
+      justify-content: end;
+      height: 120px;
+    }
+  }
+
+  .outside-closures-view {
```

Comment:
Since `current-elevator-closed-view` and `outside-closures-view` are pretty independent of each other, what do you think about splitting them into separate stylesheets? This file is getting pretty long as it is.

Reply:
Good call. Will work on splitting out the styles and React components.